src/goto-symex/goto_symex_state.h

```diff
@@ -72,13 +84,37 @@ class goto_symex_statet final
   symex_level1t level1;
   symex_level2t level2;
 
-  // Map L1 names to (L2) constants
+  // Map L1 names to (L2) constants. Values will be evicted from this map
+  // when they become non-constant. This is used to propagate values that have
+  // been worked out to only have one viable value.
+  //
+  // "constants" can include symbols, but only in the context of an address-of
+  // op (i.e. &x can be propagated), and an address-taken thing should only be
+  // L1. Actual integer literals (for example) aren't symbols, and so aren't
```

Comment:
I'd drop the mention of integer literals, it adds more confusion than it removes imo